src/Button/StyledButton.tsx

```diff
@@ -0,0 +1,15 @@
+import styled from 'styled-components';
+
+export const StyledButton = styled.button`
+  background: ${({ theme }): string => (theme.color.background.medium)};
+  border-color: ${({ theme }): string => (theme.color.background.medium)};
```

Comment:
I missed this on the first round, but we should also set `border-size` and `border-style` properties in this component so that it'll be inherited by the other buttons. I loaded the styleguide in firefox and the default button styles are a little different there than in chrome, so setting those explicitly here will make it more consistent across browsers.

Reply:
Oh true I should have included those!! Fixed through c5e3d7f.